lib/Doctrine/ORM/Mapping/AssociationOverride.php

```diff
@@ -4,13 +4,7 @@
 
 namespace Doctrine\ORM\Mapping;
 
-/**
- * This annotation is used to override association mapping of property for an entity relationship.
- *
- * @Annotation
- * @NamedArgumentConstructor
- * @Target("ANNOTATION")
- */
+/** This attribute is used to override association mapping of property for an entity relationship. */
 final class AssociationOverride implements MappingAttribute
```

Comment:
This class does not have an `#[Attribute]` attribute? Is it still needed?

Reply:
Removing it breaks the test suite. I think it's because it's only supposed to ever be used nested in an attribute, and note standalone.

Reply:
Ah, that makes sense, thanks.